### PR TITLE
Settings page

### PR DESCRIPTION
**Pull request by BrennaManning**
Admin Settings - Switching Admin status

### CODE REVIEW

File: models/user.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ var mongoose = require('mongoose'),
     Schema = mongoose.Schema,
     passportEmail = require('passport-local-mongoose-email');
 
-var User = new Schema({});
+var User = new Schema({
+	isAdmin: {type: Boolean, default: false}
```
Comment (davidabrahams) [inline models/user.js]: indent w/ spaces

**Comment by davidabrahams** [comment]
Will finish reviewing this tonight

**Comment by aconsilvio** [comment]
:cherry_blossom: :cactus: :cake: :camel: :heavy_check_mark:

**Comment by nmohamed** [comment]
:snake: ssss-super good

**Comment by davidabrahams** [comment]
Merge dis bitch.

**Pull request merged**